core/interface/static/js/skills.js

```diff
@@ -0,0 +1,202 @@
+$(document).tooltip();
+
+$(function () {
+	let selectedSkillsToDownload = [];
+	let skillStatuses = {};
+
+	function skillStatus(skill) {
+		$.ajax({
+			url: '/skills/checkInstallStatus/',
+			data: {
+				'skill': skill
+			},
+			type: 'POST'
+		}).done(function (status) {
+			skillStatuses[skill] = JSON.stringify(status).trim();
+		}).fail(function () {
+			skillStatuses[skill] = 'failed';
+		});
+	}
+
+	function checkInstallStatus(skill) {
+		skillStatus(skill);
+
+		let status = skillStatuses[skill];
+		if (status == JSON.stringify('installed')) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='. (eqeqeq)](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4699682)